README.md

```diff
@@ -41,7 +41,7 @@ const { menubar } = require('menubar');
 
 const mb = menubar();
 
-mb.on('ready', () => {
+mb.app.on('ready', () => {
```

Comment:
The previous version was correct. The 'ready' event from `menubar` is **different** from the 'ready' event from Electron's app.

The latter fires when Electron is ready (quite early in the process), and the former when the menubar tray has been created (which is officially when menubar is "ready" to be used, always after Electron app's 'ready').

I agree with you it's confusing, if you want you can modify your PR to update the README. Thanks!